### PR TITLE
Add package.json

### PR DESCRIPTION
**Pull request by thanhnhan2tn**
Add package.json to fix error install from cordova 7

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    ]
+  },
+  "keywords": [
+    "ecosystem:cordova",
+    "cordova-ios",
+    "cordova-android"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/thanhnhan2tn/phonegap-plugin-wizUtils"
```
Comment (ronkorving) [inline package.json]: Would you mind updating the repository URL to this repo? :)

**Comment by ronkorving** [comment]
Thanks for the PR! I left one tiny comment. If you could address that, we would love to merge this 👍

File: package.json (hunk 1 above)
Comment (thanhnhan2tn) [reply package.json]: That is using in my project, I cannot change it now.
You can merge and update that by your self.

**Comment by thanhnhan2tn** [comment]
Updated.

File: package.json (hunk 1 above)
Comment (thanhnhan2tn) [reply package.json]: Updated

**Comment by ronkorving** [comment]
Thanks!

**Pull request merged**